cumulusci.yml

```diff
@@ -254,7 +300,20 @@ flows:
                     include_beta: True
             5:
                 task: install_managed_beta
-
+    new_org:
```

Comment:
Looks like you based this on `feature/upgraded-org-docs`, so you've included some unrelated changes.